src/caret_analyze/plot/histogram/histogram.py

```diff
@@ -12,62 +12,51 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Sequence
+from logging import getLogger
+from typing import Optional, Sequence
 
-from bokeh.plotting import figure, show
+from bokeh.plotting import Figure
 
-from caret_analyze.exceptions import UnsupportedTypeError
-
-from caret_analyze.record import ResponseTime
-
-from .histogram_interface import HistPlot
-
-from ..plot_util import PlotColorSelector
+import pandas as pd
 
+from ..plot_base import PlotBase
+from ..visualize_lib import VisualizeLibInterface
 from ...runtime import Path
 
+logger = getLogger(__name__)
+
 
 # TODO: Migrate drawing process to visualize_lib
-class ResponseTimePlot(HistPlot):
+class ResponseTimeHistPlot(PlotBase):
 
     def __init__(
         self,
+        visualize_lib: VisualizeLibInterface,
         target: Sequence[Path],
         case: str = 'best-to-worst',
         binsize_ns: int = 10000000
```

Comment:
[suggestion]
This function appears to be called only from `ResponseTimeHistPlotFactory.create_instanse`.
In that case, it does not seem necessary for the arguments to be optional.

```suggestion
    def __init__(
        self,
        visualize_lib: VisualizeLibInterface,
        target: Sequence[Path],
        case: str,
        binsize_ns: int
```